libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Microsoft.IActivityTemplate.schema

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
Did you decide to keep the schema names as IX when they are interfaces in the schema?